homeassistant/components/nightscout/sensor.py

```diff
@@ -67,7 +67,11 @@ async def async_update(self) -> None:
                 ATTR_DELTA: value.delta,
                 ATTR_DIRECTION: value.direction,
             }
-            self._attr_native_value = value.sgv
+            """Convert to mmol/l and keep 2 decimals"""
+            if self._attr_native_unit_of_measurement == 'mmol/l':
+                self._attr_native_value = '{:.2f}'.format(value.sgv * 0.0555)
+            else:
+                self._attr_native_value = value.sgv
```

Comment:
can we recognzie the unit from the API?

Reply:
If the API always returns in `mg/dL`, then maybe it should be solved with a `device_class` and `UnitConverter` instead.

Reply:
The latest version of the api (1.3.3) does provide mmol but expects a [delta](https://github.com/marciogranzotto/py-nightscout/blob/v1.3.3/py_nightscout/models.py#L82) to return from the api that is not there in the [current add-on version](https://github.com/marciogranzotto/addon-nightscout/blob/v1.4.1/nightscout/Dockerfile#L13).

Reply:
I could try a PR on that repo but the last commit was 3 years ago

Reply:
I haven't seen the maintainer active in a long while. I am not sure if we can use a unit converter here because we don't have a device class yet and only 2 integrations where this is a problem.

Reply:
For now, let's just make sure we use constants for the units, and that these constants match the values from `dexcom` integration. It will make it easier to migrate to UnitConversion later on if more integrations need it.

Reply:
I created the constants in the const.py, I hope that's what you mean.